installer-resources/chained.yaml

```diff
@@ -1,3 +1,4 @@
+chainedservers:
```

Comment:
Without this line, `len(servers.ChainedServers)` will always be zero when reading from chained.yaml